File: annotation/src/main/java/com/cangwang/annotation/ModuleGroup.java
package com.cangwang.annotation;

import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;

/**
 *
 * Created by cangwang on 2017/8/31.
 */
@Target(ElementType.TYPE)
@Retention(RetentionPolicy.SOURCE)
public @interface ModuleGroup {
    ModuleUnit[] value();
}

File: annotation/src/main/java/com/cangwang/annotation/ModuleGroup.kt
package com.cangwang.annotation

/**
 *
 * Created by cangwang on 2017/8/31.
 */
@Target(AnnotationTarget.CLASS, AnnotationTarget.FILE)
@kotlin.annotation.Retention(AnnotationRetention.SOURCE)
annotation class ModuleGroup(vararg val value: ModuleUnit)
